src/olympia/files/utils.py

```diff
@@ -106,6 +106,12 @@ def id_to_path(pk, breadth=1):
 def get_file(fileorpath):
     """Get a file-like object, whether given a FileUpload object or a path."""
     if hasattr(fileorpath, 'path'):  # FileUpload
+        if not fileorpath.path:
+            # This upload has already been used to create a Version. Note that
+            # we say "processed" but it's different than the `processed`
+            # property on FileUpload, it's just used here as a generic term to
+            # inform the developer this FileUpload has already been used.
+            raise AlreadyUsedUpload(gettext('This upload is already being processed.'))
```

Comment:
`This upload has already been used` or `This upload has already been submitted` (or if we really want to say "processed", even though we have a `FileUpload.processed` property that refers to validation instead, `This upload has already been processed`)

Reply:
Can we add a sentence on what the developer can do about this issue? What is the next step? Wait? Proceed somewhere else? Upload a new version?

Reply:
We don't know their intent, but there is probably nothing to do on their side : the `FileUpload` has already been processed, there is likely already a `Version`. 

If we knew what they did we could be a little more specific but it's difficult to be sure. One way to reproduce, that I described in the issue, is going back after submission and trying to submit again from the same upload. Another would be to manage to quickly submit twice so that the first request gets processed before the second request goes through.